.github/workflows/release-artifacts.yml

```diff
@@ -185,3 +203,44 @@ jobs:
         GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
       run: |
         gh workflow run upload-win-installer.yml -f version=${{steps.getversion.outputs.version}} -f dryrun=false
+    outputs:
+      dryrun: ${{ steps.actual_dryrun.outputs.dryrun }}
+      version: ${{ steps.getversion.outputs.version }}
+  notification:
+    if: needs.build.outputs.dryrun == 'false'
+    runs-on: ubuntu-22.04
+    needs: build
+    steps:
+      - name: Format release email
+        env:
+          VERSION: ${{ needs.build.outputs.version }}
+        run: |
+          cat <<EOF>email_body.txt
+          Hi all,
+
+          Podman $VERSION is now available.  You may view the full details at
+          https://github.com/${{ github.repository }}/releases/tag/$VERSION
+
+          Release Notes:
+          --------------
+          EOF
+
+          echo ${{ secrets.GITHUB_TOKEN }} | gh auth login --with-token
+          gh release view $VERSION \
+            --repo ${{ github.repository }} --json=body --jq '.body' >> email_body.txt
+
+      - name: Debug release e-mail contents
```

Comment:
Same here, Looks like a debug

Reply:
Similar reasoning, the full contents of the e-mail won't be available if there's a failure.  This ensures that it's always present.  Though arguably one could simply go check what got sent to the mailing list, so really this just saves maintainers from needing to check their mail :smile: